jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/SlideGroupsTests.java

```diff
@@ -31,109 +27,74 @@ public void before() {
 
     @Test
     public void activeClassSlideGroupTests() {
```

Comment:
you know count of slides. I suggest to redesign test to work in cycle. Select first slide, check the rest of slides that they are not selected.